src/main/java/com/example/bookstoreapp/services/OrderService.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bookstoreapp.services;
+
+import com.example.bookstoreapp.models.ShoppingOrder;
+import com.example.bookstoreapp.models.OrderRequest;
+
+public interface OrderService {
```

Comment:
Cant we take OrderItem as return type.

Reply:
When we place order it has more than one items along with it has details about address where the order need to be ship.

So construct ShopingOrder class that say what are the items present in order  , what is the address, what is the total Amount, how many items in the order and so on.